core/src/test/java/de/evoila/cf/broker/controller/core/CatalogControllerTest.java

```diff
@@ -0,0 +1,136 @@
+package de.evoila.cf.broker.controller.core;
+
+import com.google.common.reflect.TypeToken;
+import com.google.gson.Gson;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+import java.io.File;
+import java.io.FileReader;
+import java.util.ArrayList;
+import java.util.List;
+
+import de.evoila.cf.broker.model.catalog.Catalog;
+import de.evoila.cf.broker.model.catalog.ServiceDefinition;
+import de.evoila.cf.broker.service.CatalogService;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+class CatalogControllerTest {
+
+    private static final String PATH_VALID_CATALOG_SERVICES = "." + File.separator +
+                                                              "src" + File.separator +
+                                                              "test" + File.separator +
+                                                              "resources" + File.separator +
+                                                              "catalogServices.json";
+    private static final String REQUEST_IDENTITY = "a82715af-3dcd-4650-bb90-dad096294e16";
+    private static final String ORIGINATING_IDENTITY = "cloudfoundry eyANCiAgInVzZXJfaWQiOiAiNjgzZWE3NDgtMzA5Mi00ZmY0LWI2NTYtMzljYWNjNGQ1MzYwIg0KfQ==";
+
+    @Mock
+    private Catalog catalog;
+    @Mock
+    private CatalogService catalogService;
+    private CatalogController catalogController;
+
+    @BeforeEach
+    void setUp() {
+        this.catalogController = new CatalogController(catalogService);
+    }
+
+    @AfterEach
+    void tearDown() {
```

Comment:
Method necessary or just added for future additions?

Reply:
Wasn't removed unintentional.